### PR TITLE
Asset List

### PR DESCRIPTION
**Pull request by KalervoHyyppa**

### CODE REVIEW

File: lib/v2/core/models/keychain.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// TODO SDK, I dont think Keychain is the proper term
```
Comment (ultimaterex) [inline lib/v2/core/models/keychain.dart:1]: This probably is just "network"

File: lib/v2/core/providers/loaded_assets_provider.dart
Comment (ultimaterex) [inline lib/v2/core/providers/loaded_assets_provider.dart:1]: I like what you're going for here

File: lib/v2/core/utils/ui_utils.dart
Comment (ultimaterex) [inline lib/v2/core/utils/ui_utils.dart:1]: We both had the same idea,  I have a similar function in  build_context_extensions.dart (context.showSnackBar)

**Review by ultimaterex — APPROVED** [review]
A  few comments, but great job so far!
Ribn 2.0 is shaping up ✨

File: lib/v2/core/utils/ui_utils.dart
Comment (KalervoHyyppa) [reply lib/v2/core/utils/ui_utils.dart:1]: Awesome, I'll merge and update

File: lib/v2/core/models/keychain.dart (hunk 1 above)
Comment (KalervoHyyppa) [reply lib/v2/core/models/keychain.dart:1]: I like that naming

**Pull request merged**